ember-toucan-core/src/components/form/input-field.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+
+interface ToucanFormInputFieldComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    label?: string;
```

Comment:
Should we make `label` required for a11y reasons? (Even if we eventually allow it to be visually hidden)

```suggestion
    label: string;
```

Reply:
I totally missed this from the notes...need to add an assert here too (and test for it)